### ISSUE TITLE
Remove coveralls badge from Readme

### ISSUE THREAD

**Issue by kaustubh-nair** [publiclab/mapknitter#648]
Hi, this is a [first-timers-only issue](https://code.publiclab.org/#r=all). This means we've worked to make it more legible to folks who either **haven't contributed to our codebase before, or even folks who haven't contributed to open source before**. 

If that's you, we're interested in helping you take the first step and can answer questions and help you out as you do. Note that we're especially interested in contributions from people from groups underrepresented in free and open source software!

We know that the process of creating a pull request is the biggest barrier for new contributors. This issue is for you 💝

If you have contributed before, **consider leaving this one for someone new**, and looking through our general [help wanted](https://github.com/publiclab/plots2/labels/help-wanted) issues. Thanks!

### 🤔 What you will need to know.

Nothing. This issue is meant to welcome you to Open Source :) We are happy to walk you through the process.

### 📋 Step by Step

- [ ] 🙋 **Claim this issue**: Comment below. If someone else has claimed it, ask if they've opened a pull request already and if they're stuck -- maybe you can help them solve a problem or move it along!

- [ ] 📝 **Update** 

In `README.md` file,  delete the first line and replace it with the following
```
## MapKnitter 2 [![codecov](https://codecov.io/gh/publiclab/mapknitter/branch/main/graph/badge.svg)](https://codecov.io/gh/publiclab/mapknitter) [![View performance data on Skylight](https://badges.skylight.io/typical/ArYnJAb3VUC9.svg?token=DJ-zenCIFAootUAeQ8BkTiTkMBXkNpNc-PXTLA4dqDU)](https://www.skylight.io/app/applications/ArYnJAb3VUC9)

```

- [ ] 💾 **Commit** your changes

- [ ] 🔀 **Start a Pull Request**. There are two ways how you can start a pull request:

1. If you are familiar with the terminal or would like to learn it, [here is a great tutorial](https://egghead.io/series/how-to-contribute-to-an-open-source-project-on-github) on how to send a pull request using the terminal.

2. You can also [edit files directly in your browser](https://help.github.com/articles/editing-files-in-your-repository/) and open a pull request from there. 

- [ ] 🏁 **Done** Ask in comments for a review :)

### 🤔❓ Questions?

Leave a comment below!

### Is someone else already working on this?

We encourage you to link to this issue by mentioning the issue # in your pull request, so we can see if someone's already started on it. **If someone seem stuck, offer them some help!** Otherwise, [take a look at some other issues you can help with](https://code.publiclab.org/#r=all). Thanks!

**Comment by st3v0o** [comment]
I would like to claim this issue. Thanks!

**Comment by ManethKulatunge** [comment]
I would like to claim this if available.

**Comment by kaustubh-nair** [comment]
@st3v0o @ManethKulatunge I believe @garg3133 already has a PR on this one!
Would it be okay if you try another FTO issue?
Thanks!

**Comment by sashadev-sky** [comment]
@ManethKulatunge are you still looking for an FTO? If so you can add your name to the FTO issues tracker in https://github.com/publiclab/plots2/issues/4963 and say you want something for Mapknitter specifically and we'll assign you. 

Or you can grab this open FTO from the leaflet-environmental-layers repo: publiclab/leaflet-environmental-layers#177

**Comment by sashadev-sky** [comment]
closed via #652
